### ISSUE TITLE
Running without port 80/443

### ISSUE THREAD

**Issue by CountParadox** [subspacecloud/subspace#34]
I am attempting to run this without binding to port 80/443 on my host, as these ports are already in use by nginx. I can do my SSL and reverse proxy from there, and do not want the internal one with subspace. 

I attempted to run the container without the --cap-add NET_ADMIN \ line, and on the bridge network, rather than the host, however I am unable to connect to the admin interface on a mapped port to 443 in the container.

Is this the correct way to do this, and if not, how would I go about starting this up without binding to port 443 / 80 on the host?

**Comment by Semetra22** [comment]
This worked for me:

`docker create --name subspace --restart always --network host --cap-add NET_ADMIN --volume /usr/bin/wg:/usr/bin/wg --volume /data:/data --env SUBSPACE_HTTP_HOST=host.example.com --env SUBSPACE_HTTP_ADDR=":8888" --env SUBSPACE_HTTP_INSECURE="true" subspacecloud/subspace:latest`

Just enter the HTTP-Port (in my case 8888) you want and you're ready to go.

**Comment by ovizii** [comment]
@Semetra22 
If I am not mistaken your solution would expose subspace on port 8888 to the internet, correct?

**Comment by Semetra22** [comment]
@ovizii Thats correct. You can change the port exposed by changing `SUBSPACE_HTTP_ADDR=":8888"`
to the port you would like.

**Comment by ovizii** [comment]
thanks. let me add a question, maybe you can help.

I would like to have subspace behind a reverse proxy so I assume I  cannot run it with `--network host` 
Has anyone succeeded doing so? That is also what the initial poster asked for. Your solution sidesteps the port problem but I would still like to get it working behind a reverse proxy.

**Comment by CountParadox** [comment]
I moved to using Pritunl inside its own VM in the end :(
